src/server.js

```diff
@@ -39,7 +37,11 @@ function getQtumProcess() {
 // Checks to see if the qtumd port is still in use
 function checkQtumPort() {
   const port = isMainnet() ? Config.RPC_PORT_MAINNET : Config.RPC_PORT_TESTNET;
-  portscanner.checkPortStatus(port, Config.HOSTNAME, (error, status) => {
+  portscanner.checkPortStatus(port, Config.HOSTNAME, (err, status) => {
+    if (err) {
+      getLogger().error(`Error: qtumd: ${err.message}`);
+      throw err;
```

Comment:
remove the throw. This is specifically for when the user uses the `Launch Qtum Wallet` option in the dapp. We check the port continuously to see when we can start the Qtum Wallet since qtumd and qtum-qt can't run simultaneously. throwing here would break the check. let's just fail silently.

Reply:
I see, make sense.